mandatory/2-function-creation.js

```diff
@@ -6,7 +6,9 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  return (typeof num ==="number") && ( num % 2=== 0 ) && (num<=100) ;
```

Comment:
Really nice how you broke up each check with brackets here

It's not necessary for the code to work, but it's handy for the reader, and code should always be written for humans to read. 🥇